File: Webapp/webApp/src/main/java/com/project/webapp/Entity/Doctor.java
package com.project.webapp.Entity;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.Date;


@Data
@NoArgsConstructor
@AllArgsConstructor
@Document(collection = "doctor")
@JsonInclude(JsonInclude.Include.ALWAYS)
public class Doctor {
    @Id
    private String id;
    private String email;
    private String firstName;
    private String lastName;
    private String timeAvailabe;
    private String description;
    private String specializing;
    private int status;
    private double amount;
    private int cured;
    private String venue;
    private String added;
    private Date edited;
}

File: frontend/IcareMobile/app/src/main/java/com/example/myapplication/Entity/Doctor.kt
package com.example.myapplication.Entity

import java.io.Serializable

data class Doctor(
    var  email:String,
    var  firstName:String,
    var  lastName:String,
    var  timeAvailabe:String,
    var description:String,
    var specializing:String,
    var status:Int,
    var amount:Double,
    var cured:Int,
    var venue:String,
    var added:String
):Serializable
